### PR TITLE
Fixes #9139 - restore compatibility with latest networking

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

File: snippets/kickstart_networking_setup.erb [hunk 1]
```diff
@@ -113,8 +116,14 @@ DEVICE="$real"
 HWADDR="<%= interface.mac -%>"
 <% end -%>
 ONBOOT=yes
+<% if interface.respond_to?(:primary) && interface.primary -%>
+DEFROUTE=yes
```
Comment (domcleal) [inline snippets/kickstart_networking_setup.erb]: this is a PPP-specific option, I don't think it's relevant as the route'd be determined by the gateway?

Comment (ares) [inline snippets/kickstart_networking_setup.erb]: I think that default route is usually determined by interface gateway which comes from DHCP. We can have more interfaces, each with different gateway. My understanding was that in such case DEFROUTE sets which gateway is used for default route (making it default gateway). I'm not 100% sure about all DEFROUTE impacts and there's not much documentation.

**Comment by domcleal** [comment]
Looks about right to me :+1:

**Comment by mmoll** [comment]
I didn't test this, but looking OK, so merged. Thanks @ares!

**Pull request merged**